tools/integration-tests/tester/framework/parameters.go

```diff
@@ -42,6 +42,8 @@ var (
 	ParaPoWFaucetDifficulty = 2
 	// ParaSyncBeaconOnEveryNode defines whether all nodes should be sync beacons.
 	ParaSyncBeaconOnEveryNode = false
+	// ParaManaOnEveryNode defines whether all nodes should have mana enabled.
+	ParaManaOnEveryNode = false
```

Comment:
```suggestion
	ParaManaOnEveryNode = true
```
mana is an essential component which is enabled by default on a node, why wouldn't we enable it then on all nodes in tests?